src/layers/event/versioning/constants.py

```diff
@@ -2,5 +2,8 @@
 
 VERSION_HEADER_PATTERN = r"^(\d+)$"
 VERSION_RE = re.compile(r"^v(\d+)$")
-API_ROOT_DIRNAME = "api"
-VERSIONED_HANDLER_GLOB = "src/v*/steps.py"
+
+
+class VERSIONING_STEP_ARGS:
+    VERSIONED_STEPS = "versioned_steps"
```

Comment:
Related to the comment above where this default doesnt match the object of {"1": v1_steps}

Reply:
I know you type enforce it in the functions, just a suggestion to make that a bit more secure